### PR TITLE
Add QueryDisks

### PR DESCRIPTION
**Pull request by vbauzysvmware**
* Add QueryDisks using name
Function needed to handle independent disk read/import by Id

### CODE REVIEW

File: govcd/disk.go [hunk 1]
```diff
@@ -350,6 +350,31 @@ func (vdc *Vdc) QueryDisk(diskName string) (DiskRecord, error) {
 	return *newDisk, nil
 }
 
+// Find independent disks using disk name. Returns list of VMRecord query type
+func (vdc *Vdc) QueryDisks(diskName string) (*[]*types.DiskRecordType, error) {
+
+	if diskName == "" {
+		return nil, fmt.Errorf("disk name can not be empty")
```
Comment (dataclouder) [inline govcd/disk.go]: "can not" => "can't"

File: govcd/disk.go [hunk 2]
```diff
@@ -350,6 +350,31 @@ func (vdc *Vdc) QueryDisk(diskName string) (DiskRecord, error) {
...
+	}
+
+	typeMedia := "disk"
+	if vdc.client.IsSysAdmin {
+		typeMedia = "adminDisk"
+	}
+
+	results, err := vdc.QueryWithNotEncodedParams(nil, map[string]string{"type": typeMedia, "filter": "name==" + url.QueryEscape(diskName)})
+	if err != nil {
+		return nil, fmt.Errorf("error querying disk %#v", err)
```
Comment (dataclouder) [inline govcd/disk.go]: "%#v" => "%s"

File: govcd/disk.go [hunk 3]
```diff
@@ -350,6 +350,31 @@ func (vdc *Vdc) QueryDisk(diskName string) (DiskRecord, error) {
...
+	}
+
+	diskResults := results.Results.DiskRecord
+	if vdc.client.IsSysAdmin {
+		diskResults = results.Results.AdminDiskRecord
```
Comment (dataclouder) [inline govcd/disk.go]: What's the difference between `DiskRecord` and `AdminDiskRecord`? 
Does the caller need to know?

File: govcd/disk.go (hunk 1 above)
Comment (vbauzysvmware) [reply govcd/disk.go]: updated

File: govcd/disk.go (hunk 2 above)
Comment (vbauzysvmware) [reply govcd/disk.go]: updated

File: govcd/disk.go (hunk 3 above)
Comment (vbauzysvmware) [reply govcd/disk.go]: It is how you should call and what type receive to/from Query with different type of users `system` -`none system`. I made it that way that user don't know the details.

File: govcd/disk.go [hunk 4]
```diff
@@ -350,6 +350,31 @@ func (vdc *Vdc) QueryDisk(diskName string) (DiskRecord, error) {
 	return *newDisk, nil
 }
 
+// Find independent disks using disk name. Returns list of VMRecord query type
```
Comment (Didainius) [inline govcd/disk.go]: go doc format
```suggestion
// QueryDisks ...
```

Comment (vbauzysvmware) [reply govcd/disk.go]: tx. Updated

**Pull request merged**